scala/pom.xml

```diff
@@ -1,10 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.microsoft.spark</groupId>
-  <artifactId>spark-clr</artifactId>
+  <groupId>com.microsoft.sparkclr</groupId>
+  <artifactId>spark-clr_2.10</artifactId>
   <version>1.5.2-SNAPSHOT</version>
-  <name>${project.artifactId}</name>
+  <name>CLR for Apache Spark</name>
```

Comment:
IMO, "CLR for Apache Spark" could have various interpretations. Could we just use something like "SparkCLR" or "SparkCLR Project"? Description anyways provide more info on the project.